docs/installation.rst

```diff
@@ -66,46 +77,39 @@ for installing packages into your home directory. ::
 
     $ pip install webbpsf --user
 
-.. admonition:: For STScI Users Only
-
-   Users at STScI may also access WebbPSF through the standard `SSB software distributions <http://ssb.stsci.edu/ssb_software.shtml>`_. Set the environment variable ``WEBBPSF_PATH`` to ``/grp/jwst/ote/webbpsf-data``, select the desired version of SSB (``ssbx``, ``ssbdev``, or ``ssbrel``), and the webbpsf package will be available.
-
-The source code is hosted in `this repository on GitHub <https://github.com/mperrin/webbpsf>`_. See :ref:`below <install_dev_version>` for more on how
-to install from the development source.
-
+The source code is hosted in `this repository on GitHub <https://github.com/mperrin/webbpsf>`_. See :ref:`below <install_dev_version>` for more on how to install from the development source.
 
 .. warning::
-  If you get the message ``SystemError: Cannot compile 'Python.h'. Perhaps you need to install python-dev|python-devel.`` during install *even when Python.h is available*, this means ``setup.py`` was unable to install NumPy. This can sometimes be fixed by executing ``pip install numpy`` separately, before installing webbpsf. See the bug report at `numpy/numpy#2434 <https://github.com/numpy/numpy/issues/2434>`_ for details.
+  If you get the message ``SystemError: Cannot compile 'Python.h'. Perhaps you need to install python-dev|python-devel.`` during install *even when Python.h is available*, this means ``setup.py`` was unable to install NumPy. This can sometimes be fixed by executing ``pip install numpy`` separately, before installing WebbPSF. See the bug report at `numpy/numpy#2434 <https://github.com/numpy/numpy/issues/2434>`_ for details.
 
 .. _pysynphot_install:
 
 Installing or updating pysynphot
----------------------------------
+--------------------------------
 
 Pysynphot is an optional dependency, but is highly recommended.  Installation instructions can be found `here in the POPPY docs <http://pythonhosted.org//poppy/installation.html#installing-or-updating-pysynphot>`_.
 
 .. _data_install:
 
 Installing the Required Data Files
----------------------------------------------
+----------------------------------
 
 Files containing such information as the JWST pupil shape, instrument throughputs, and aperture positions are distributed separately from WebbPSF. To run WebbPSF, you must download these files and tell WebbPSF where to find them using the ``WEBBPSF_PATH`` environment variable.
 
-1. Download the following file:  `webbpsf-data-0.4.1.tar.gz <http://www.stsci.edu/~mperrin/software/webbpsf/webbpsf-data-0.4.1.tar.gz>`_  [approx. 430 MB]
-2. Untar ``webbpsf-data-0.4.1.tar.gz`` into a directory of your choosing.
+1. Download the following file:  `webbpsf-data-0.5.0.tar.gz <http://www.stsci.edu/~mperrin/software/webbpsf/webbpsf-data-0.5.0.tar.gz>`_  [approx. 430 MB]
```

Comment:
I should add it to https://github.com/mperrin/webbpsf/blob/master/docs/release.rst !